playbooks/rhv/inventory.example

```diff
@@ -0,0 +1,99 @@
+[all:vars]
+app_dns_prefix=apps
+public_hosted_zone=example.com
+load_balancer_hostname=lb0.{{public_hosted_zone}}
+openshift_master_cluster_hostname="{{ load_balancer_hostname }}"
+openshift_master_cluster_public_hostname=openshift-master.{{ public_hosted_zone }}
+openshift_master_default_subdomain="{{ app_dns_prefix }}.{{ public_hosted_zone }}"
+openshift_public_hostname="{{openshift_master_cluster_public_hostname}}"
+
+[OSEv3:children]
+nodes
+masters
+etcd
+lb
+
+[OSEv3:vars]
+# General variables
+ansible_ssh_user=root
```

Comment:
Please add to the vars in parentheses when that is the default value.

Reply:
Like: ansible_ssh_user=root (Default)

Reply:
Or just have them commented out
like:
# ansible_ssh_user=root

Reply:
Also, I see that it was deprecated in favor of ansible_user

Reply:
Ah, I'll update to look like: https://github.com/openshift/openshift-ansible/blob/master/inventory/hosts.example#L39